src/lib/zxing/index.ts

```diff
@@ -0,0 +1,26 @@
+import { BrowserCodeReader, BrowserQRCodeReader, IScannerControls } from '@zxing/browser'
```

Comment:
Use dynamic import. It's important to lazy load `@zxing/browser` due big bundle size. Check how it was done before refactoring.